code/controllers/subsystem/economy.dm

```diff
@@ -37,6 +37,13 @@ SUBSYSTEM_DEF(economy)
 	var/earning_report
 	var/market_crashing = FALSE
 
+	/// Total value of exported materials.
+	var/export_total = 0
+	/// Total value of imported goods.
+	var/import_total = 0
```

Comment:
These vars are only used to increment when exports and imports happen and arent actually used for anything

Reply:
The PR was made prematurely by this third party. I was considering making mail generate based on cargo activity.